conans/client/manager.py

```diff
@@ -474,6 +474,11 @@ def build(self, conanfile_path, source_folder, build_folder, package_folder, ins
             except ConanException:
                 pass
 
+        if should_configure or should_build or should_install:
```

Comment:
if I pass three "False" it should do nothing, I know it is kind of ridiculous use case, but I think you should check if any is not None to activate the "should mode" to be consistent with the "None" meaning.

Reply:
And if any of the three is not None, adjust individually to False when None:

```python
if should_configure is not None or should_build is not None or should_install is not None:
    conanfile.should_configure = bool(should_configure)
    conanfile.should_build = bool(should_build)
    conanfile.should_install = bool(should_install)
```

Reply:
I have thinking about this, and have one question. What would be the behavior of passing in the API:

```python
build(should_configure=False, should_build=None, should_install=None)
build(should_configure=None, should_build=False, should_install=None)
...
build(should_configure=True, should_build=None, should_install=None)
```

Maybe the API is wrong, actually the default value in the API should be ``True`` as it is the current default behavior, isn't it? Then it would just be ``command.py`` who would adapt the args accordingly.

Reply:
Submitted a proposal with it, I think it is a bit clearer, but let me know otherwise.

Reply:
Yes. even better. good work